internal/command/container.go

```diff
@@ -175,6 +173,10 @@ func goBuild(ctx Context, image containerImage) error {
 	// add go build command
 	args := []string{"go", "build", "-trimpath"}
 
+	if ctx.BuildMode != "" {
```

Comment:
This will need to be rebased on develop and be added to GOFLAGS instead as that's how you can pass flags to the go command when using fyne build.

Reply:
Merged develop in 7c2f186